.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }} ${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
+      - name: Code Review Agent - Issue Comment
+        if: github.event_name == 'issue_comment'
+        uses: gitbito/codereviewagent@main
+        with:
+          pr: ${{ github.event.issue.pull_request.html_url }}
+          command: ${{ github.event.comment.body }}
+          options: ${{ env.OPTIONS }}
+
+      - name: Set Options Environment Variable for Pull Request
+        if: github.event_name == 'pull_request'
+        run: |
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }} ${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
+      - name: Code Review Agent action step
+        if: github.event_name == 'pull_request'
+        uses: gitbito/codereviewagent@main
+        with:
+          pr: ${{ github.event.pull_request.html_url }}
+          command: review
+          options: ${{ env.OPTIONS }}
```

Comment:
**Code Structure Issue**: The workflow file lacks comments explaining the purpose of each job and step, which can make understanding the workflow's intention difficult for new contributors. <br> **Fix**: Add comments before each job and critical step to explain their purpose and how they contribute to the workflow. <br> **Code Suggestion**: 
 ```
 +# Define the purpose of the workflow
 +# This workflow handles pull requests and issue comments
 +on:
 +  pull_request:
 +    types: [opened, reopened, ready_for_review, review_requested]
 +  issue_comment:
 +    types: [created]
 +
 +# Job: Code review job
 +# Description: This job runs the code review agent on every pull request and responds to user comments
 +jobs:
 +  code_review_job:
 +    runs-on: ubuntu-latest
 +    permissions:
 +      issues: write
 +      pull-requests: write
 +      contents: write
 +    name: Run code review agent on every pull request, respond to user comments
 +
 +# Steps for setting environment variables and running the code review agent
 +    steps:
 ```